src/io/flutter/view/InspectorPanel.java

```diff
@@ -538,16 +543,39 @@ public void showProperties(DiagnosticsNode diagnostic) {
           return;
         }
 
-        properties.sort(Comparator.comparing(DiagnosticsNode::getName));
+        if (properties.size() == 0) {
+          // TODO(jacobr): display message that there are no properties
+          // and that no errors occurred.
```

Comment:
IntelliJ does have a convention for this - just putting a label (like "no properties" or similar) centered in the panel:

<img width="541" alt="screen shot 2017-12-07 at 7 11 23 am" src="https://user-images.githubusercontent.com/1269969/33722112-f22b1a7c-db1d-11e7-9c19-e8c4b3ee75da.png">